<ui version="4.0" >
 <class>aboutDlg</class>
 <widget class="QDialog" name="aboutDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="1" >
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>114</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>151</width>
       <height>86</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>150</width>
       <height>86</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>114</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <spacer name="verticalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Operating System:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="osLbl" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>GNU/Linux</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Processor:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="vendorLbl" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Processor vendor</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="modelLbl" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Processor model</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="speedLbl" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Processor speed/freq</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Memory:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Author:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Licence:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Credits:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QLabel" name="label_2" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>GPL2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLabel" name="label_10" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Cvetoslav Ludmiloff</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QLabel" name="label_11" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>&lt;ludmiloff@gmail.com></string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" >
      <widget class="QLabel" name="label_13" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Matteo Bertozzi, Tomasz Ziobrowski</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" >
      <widget class="QLabel" name="label_12" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Giuseppe Cigala</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLabel" name="memoryLbl" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>all available memory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>114</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>151</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>150</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../theme/std.qrc" >:/icons/antico.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Antico Deluxe</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="versionLbl" >
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>ver. ....</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../theme/std.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>aboutDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>175</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel" >
     <x>175</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
